release-irma-frontend.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "Preparing irma-frontend for release"
```

Comment:
Why is this script called `release-irma-frontend.sh`, if it does not release anything and here mentions only preparation?

Reply:
Yeah as I said above it would indeed be better to name this `prepare-irma-frontend.sh`.